6.X/6.7.0/docs/04-advanced_configuration.md

```diff
@@ -0,0 +1,211 @@
+
+```bash
+eksctl create cluster \
+   --name=eks-installer1 \
+   --node-type=m5.4xlarge \
+   --nodes=3 \
+   --version 1.14 \
+   --region=us-east-1 \
+   --vpc-public-subnets=<subnet1,subnet2>
+```
+
+### Additional Installer Configurations
+
+EKS uses aws-iam-authenticator to authorize kubectl commands.
+aws-iam-authenticator needs aws credentials mounted from **~/.aws** to the installer.
+
+```bash
+docker run  \
+  -v ~/.aws:/.aws \
+  -e HOST_USER=$(id -u) \
+  -e KUBECONFIG=/.kube/config \
+  -v ~/.kube:/.kube:Z \
+  -v $(pwd):/manifests:Z \
+  quay.io/sysdig/installer:<InstallerVersion>
+```
+
+### Running Airgapped EKS
+
+```bash
+EKS=true bash sysdig_installer.tar.gz
+```
+
+The above ensures the `~/.aws` directory is correctly mounted for the airgap
+installer container.
+
+### Exposing the Sysdig endpoint
+
+Get the external ip/endpoint for the ingress service.
+
+```bash
+kubectl -n <namespace>  get service haproxy-ingress-service
+```
+
+In route53 create an A record with the dns name pointing to external ip/endpoint.
+
+### Gotchas
+
+Make sure that subnets have internet gateway configured and has enough ips.
+
+## Airgapped Installations
+
+### Updating the feeds database in airgapped environments [ScanningV2]
+
+This is a script that can be used to automatically update the vulnerability feeds used by the ScanningV2 engine.
+
+```bash
+#!/bin/bash
+QUAY_USERNAME="<change_me>"
+QUAY_PASSWORD="<change_me>"
+
+# Calculate the tag of the last version.
+epoch=`date +%s`
+IMAGE_TAG=$(( $epoch - 86400 - $epoch % 86400))
+
+# Download image
+docker login quay.io/sysdig -u ${QUAY_USERNAME} -p ${QUAY_PASSWORD}
+docker image pull quay.io/sysdig/airgap-vuln-feeds:${IMAGE_TAG}
+# Save image
+docker image save quay.io/sysdig/airgap-vuln-feeds:${IMAGE_TAG} -o airgap-vuln-feeds-latest.tar
+# Optionally move image
+mv airgap-vuln-feeds-latest.tar /var/shared-folder
+# Load image remotely
+ssh -t user@airgapped-host "docker image load -i /var/shared-folder/airgap-vuln-feeds-latest.tar"
+# Push image remotely
+ssh -t user@airgapped-host "docker tag airgap-vuln-feeds:${IMAGE_TAG} airgapped-registry/airgap-vuln-feeds:${IMAGE_TAG}"
+ssh -t user@airgapped-host "docker image push airgapped-registry/airgap-vuln-feeds:${IMAGE_TAG}"
+
+# Update the image
+ssh -t user@airgapped-host "kubectl -n sysdigcloud set image deploy/sysdigcloud-scanningv2-airgap-vuln-feeds airgap-vuln-feeds=airgapped-registry/airgap-vuln-feeds:${IMAGE_TAG}"
+```
+
+The above script could be scheduled using a cron job that run every day like
+
+```bash
+0 8 * * * airgap-vuln-feeds-image-update.sh >/dev/null 2>&1
+```
+
+### Updating the Feeds Database in Airgapped Environments [Legacy Scanning]
+
+This is a procedure that can be used to automatically update the feeds database:
+
+1. download the image file quay.io/sysdig/vuln-feed-database-12:latest from Sysdig registry to the jumpbox server and save it locally
```

Comment:
Capitalize the first letter. Whenever you begin a sentence capitalize the first letter of the first word.

Reply:
Done